applications/nrf5340_audio/src/bluetooth/bt_renderer/volume/bt_rend_vol_internal.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BT_REND_VOL_INTERNAL_H_
+#define _BT_REND_VOL_INTERNAL_H_
+
+#include <zephyr/bluetooth/conn.h>
+
+/**
+ * @brief	Set volume to a specific value.
+ *
+ * @param	volume	The absolute volume to set.
+ *
+ * @return	0 for success,
+ *		-ENXIO if the feature is disabled,
+ *		other errors from underlying drivers.
```

Comment:
```suggestion
 * @retval	0           Volume change success.
 * @retval 	-ENXIO    The feature is disabled.
 * @retval 	other       Errors from underlying drivers.
```
Global edit for this file.